test/Microsoft.AspNetCore.Diagnostics.Tests/DeveloperExceptionPageMiddlewareTest.cs

```diff
@@ -27,6 +28,42 @@ namespace Microsoft.AspNetCore.Diagnostics
 {
     public class DeveloperExceptionPageMiddlewareTest
     {
+        public static TheoryData GetMethodDisplayString_ReturnsTypeNameQualifiedMethodsData
+        {
+            get
+            {
+                var thisType = typeof(DeveloperExceptionPageMiddlewareTest);
+                var intParse = ((Func<string, int>)int.Parse).GetMethodInfo();
+                var dateTimeOffsetTryParse = typeof(DateTimeOffset).GetMethods()
+                    .First(m => m.Name == nameof(DateTimeOffset.TryParse) && m.GetParameters().Length == 2);
+                var genericTypeMethod = typeof(List<Process>).GetMethod(nameof(List<Process>.Remove));
+                var genericMethod = thisType.GetMethod(nameof(GenericMethod));
+                var byRefMethod = thisType.GetMethod(nameof(ByRefMethod));
+                var asyncMethod = thisType.GetMethod(nameof(AsyncMethod));
+
+                return new TheoryData<MethodBase, string>
+                {
+                    { intParse, "int System.int.Parse(string s)" },
+                    { dateTimeOffsetTryParse, "bool System.DateTimeOffset.TryParse(string input, out System.DateTimeOffset& result)" },
```

Comment:
They do. I think it might be a lot more useful once your stack trace contains several different components (Mvc, Configuration, your code's Startup) rather than when it's viewed in isolation like so.